src/sound/customaudio.js

```diff
@@ -0,0 +1,82 @@
+const audioPlayer = document.querySelector('.audio-player-custom');
```

Comment:
ok i'm going to change it now

Reply:
i'm going to change the file  name with skin.js and the class skin-audio-player instead of audio-player-custom ,
i will change the name of css file too